### ISSUE TITLE
The internal HSL calculation is incorrect.

### ISSUE THREAD

**Issue by cr-jr** [quarksuite/core#3]
The current HSL calculation formula missed the conversion of the RGB channel input.

That is, it's missing **this** part: 

```js
// hslCalc(r: number, g: number, b: number): number[]
const R = r / 255;
const G = g / 255;
const B = b / 255;
```
So the output is wrong in the current version. Aaand in a few previous versions... :neutral_face: 

It's such a basic calculation error that I'm a little peeved. It will be fixed—among other issues—over the next few days.

Thanks for your patience.

**Comment by cr-jr** [comment]
Ah, I see. The conversion happens in the `rgb2HSL` function. **After** which `hslCalc` is called.

```js
// RGB -> HSL
export const rgb2Hsl = (rgb: string) => {
  const data = rgbData(rgb);
  // Make RGB channels fractions of 1
  let r = +data[0] / 255;
  let g = +data[1] / 255;
  let b = +data[2] / 255;

  const hsl = hslCalc(r, g, b);

  return `hsl(${isNaN(hsl[0]) ? 0 : hsl[0]}, ${hsl[1]}%, ${hsl[2]}%)`;
};
```

Still, that's the wrong place for the conversion to happen, so I'm still considering this an issue.

**Comment by cr-jr** [comment]
Resolved in v2.3.0
